src/components/PaymentCard.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Typography, Button } from '@material-ui/core';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      border: '1px solid',
+      borderColor: '#F7F9FC',
+      borderRadius: '6px',
+      backgroundColor: '#F7F9FC',
+    },
+    content: {
+      color: theme.palette.text.primary,
+      padding: '15px 20px',
+    },
+    itemWrapper: {
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+    },
+    break: {
+      height: '20px',
+    },
+    button: {
+      borderRadius: '12px',
+      width: '120px',
+      color: 'white',
+      backgroundColor: theme.palette.primary.main,
+    },
+    paymentText: {
+      color: theme.palette.primary.main,
```

Comment:
this styling can be removed and can be assigned with `color="primary"` in the component